common/contenttypes.go

```diff
@@ -109,6 +112,35 @@ func (c ContentTypes) RemoveOverride(path string) {
 	}
 }
 
+// RemoveOverrideByIndex removes an override given a path and override index.
+func (c ContentTypes) RemoveOverrideByIndex(path string, indexToFind int) {
+	pathPrefix := path[0:len(path)-5] // cut off '0.xml' from the end
+	if !strings.HasPrefix(pathPrefix, "/") {
+		pathPrefix = "/" + pathPrefix
+	}
+
+	re := regexp.MustCompile(pathPrefix + "([0-9]+).xml")
```

Comment:
This is risky since this will panic if it is not a valid regexp.  Maybe just use regexp.Compile and if there is an error return an error?

Reply:
ok, will do

Reply:
fixed